service/identity.go

```diff
@@ -92,3 +93,79 @@ func (s *service) Probe(
 	log.Debugf(fmt.Sprintf("Probe returning: %v", rep.Ready.GetValue()))
 	return rep, nil
 }
+
+func (s *service) GetReplicationCapabilities(ctx context.Context, req *csiext.GetReplicationCapabilityRequest) (*csiext.GetReplicationCapabilityResponse, error) {
+	var rep = new(csiext.GetReplicationCapabilityResponse)
+	if !strings.EqualFold(s.mode, "node") {
+		rep.Capabilities = []*csiext.ReplicationCapability{
+			{
+				Type: &csiext.ReplicationCapability_Rpc{
+					Rpc: &csiext.ReplicationCapability_RPC{
+						Type: csiext.ReplicationCapability_RPC_CREATE_REMOTE_VOLUME,
+					},
+				},
+			},
+			{
+				Type: &csiext.ReplicationCapability_Rpc{
+					Rpc: &csiext.ReplicationCapability_RPC{
+						Type: csiext.ReplicationCapability_RPC_CREATE_PROTECTION_GROUP,
+					},
+				},
+			},
+			// {
```

Comment:
Please add a note here or in PR description as to why this is commented so that anyone picking it up has a starting point.

Reply:
This part alongside supported actions will be uncommented in further PRs that will add support for actions execution and protection group monitoring. This PR does not implement those calls yet they just left as commented out for now